src/lang/inspect/usages.rs

```diff
@@ -98,6 +96,28 @@ impl<'a> FindUsages<'a> {
         files.into_iter()
     }
 
+    fn match_offsets<'b>(
```

Comment:
[nit]
```suggestion
    fn offsets_of_text_matches<'b>(
```
or `text_matches_offsets`. `match_offsets` suggests it matches some offsets

Reply:
I prefer my form, because `match` is a verb here that states what action is done